piet-coregraphics/src/text.rs

```diff
@@ -220,6 +220,10 @@ impl CoreGraphicsTextLayoutBuilder {
     fn current_font(&self) -> CTFont {
         //TODO: this is where caching would happen, if we were implementing caching;
         //store a tuple of attributes resolved to a generated CTFont.
+
+        // 'wght' as an int
+        const WEIGHT_AXIS_ID: i32 = 2003265652;
```

Comment:
This would be easier to read in hex (0x77676874), but even easier to read as `tag("wght")`, which is now viable due to const fn improvements:

```rust
const fn tag(s: &str) -> u32 {
    let b = s.as_bytes();
    ((b[0] as u32) << 24) | ((b[1] as u32) << 16) | ((b[2] as u32) << 8) | (b[3] as u32)
}
```